analyzer/src/main/kotlin/managers/Sbt.kt

```diff
@@ -213,3 +214,17 @@ class Sbt(
         // This is not implemented in favor over overriding [resolveDependencies].
         throw NotImplementedError()
 }
+
+private fun moveGeneratedPom(pomFile: File): File {
+    val targetDir = pomFile.absoluteFile.parentFile.searchUpwardsForSubdirectory("target")
```

Comment:
Nit: This is not the "targetDir", but the "targetDirParent".

Reply:
Done.